tests/contrib/scrapy/test_pipeline_cerberus.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import absolute_import
+from scrapy.utils.test import get_crawler
+from scrapy import Item
+
+import pytest
+from spidermon.contrib.scrapy.pipelines import ItemValidationPipeline
+from tests.fixtures.items import TestItem, TreeItem
+from tests.fixtures.validators import (
+    cerberus_tree_schema,
+    cerberus_test_schema,
+    cerberus_error_test_schema,
+)
+
+STATS_AMOUNTS = "spidermon/validation/validators"
+STATS_ITEM_ERRORS = "spidermon/validation/items/errors"
+STATS_MISSINGS = "spidermon/validation/fields/errors/missing_required_field"
+STATS_TYPES = "spidermon/validation/validators/{}/{}"
+
+SETTING_CERBERUS = "SPIDERMON_VALIDATION_CERBERUS"
+
+
+def assert_in_cerberus_stats(obj):
+    return "'{}' in {{stats}}".format(
+        STATS_TYPES.format(obj.__name__.lower(), "cerberus")
+    )
+
+
+@pytest.mark.parametrize(
+    "item,settings,cases",
+    [
+        pytest.param(
+            TestItem({"url": "example.com"}),
+            {SETTING_CERBERUS: [cerberus_test_schema]},
+            [
+                "'{}' not in {{stats}}".format(STATS_ITEM_ERRORS),
+                "{{stats}}['{}'] is 1".format(STATS_AMOUNTS),
+                assert_in_cerberus_stats(Item),
+            ],
+            id="processing usual items without errors",
+        ),
+        pytest.param(
+            TreeItem(),
+            {SETTING_CERBERUS: [cerberus_tree_schema]},
+            "'{}' in {{stats}}".format(STATS_MISSINGS),
+            id="missing required fields",
+        ),
+    ],
+)
+def test_get_crawler_only(item, settings, cases):
+    crawler = get_crawler(settings_dict=settings)
+    pipe = ItemValidationPipeline.from_crawler(crawler)
+    pipe.process_item(item, None)
+    kwargs = {"stats": "pipe.stats.stats.get_stats()"}
+    for case in cases:
+        pass
+        # FIXX THISS
```

Comment:
If you want to add a comment, add a comment to the PR.
Do not push the code with these comments

Reply:
I apologize again regarding this, I have already removed all my comments in the latest commit in my local. I take extra care now to not push any comments that are not needed. Will get rid of it.